client/my-sites/sidebar/sidebar.jsx

```diff
@@ -731,7 +694,7 @@ export class MySitesSidebar extends Component {
 		}
 
 		const publish = !! this.publish(),
-			appearance = ( !! this.themes() || !! this.menus() ),
+			appearance = ( !! this.themes() ),
```

Comment:
You may just want to [inline](https://github.com/Automattic/wp-calypso/pull/12327/files#diff-3e29fa10a6f1fb8927b10336ff9ff66bR733) this now.